website/pages/docs/builders/vmware/vsphere-clone.mdx

```diff
@@ -280,7 +280,7 @@ Minimal example of usage:
 </Tab>
 </Tabs>
 
-## Working with Clusters
+## Working with Clusters And Hosts
```

Comment:
Nit on keeping the title consistent. The anchor is "Working With Clusters And Hosts" but the title has lowercase with.